src/general/utils/TranslationHelper.cs

```diff
@@ -81,6 +81,11 @@ public static string TranslateFeatureFlag(bool enabled)
             TranslationServer.Translate("FEATURE_DISABLED");
     }
 
+    public static string TranslateBoolean(bool value)
```

Comment:
I think this might be a bit too general of a thing, above the feature flag translation was added with that specific name to help languages where true/false need to fit the context. I kind of expect a similar thing might be needed here, but I can't say yet for sure as I didn't yet read where this is used.

Reply:
I mean it could maybe be something like "PREREQUISITE_REQUIRED" and "PREREQUISITE_NOT_REQUIRED"? I think that would have enough of the context info in it to have accurate translations but still be able to use it for similar precondition required / not required display. Or maybe the icons from the crafting system for having enough of something could be used? That's accessible as `GUICommon.RequirementFulfillmentIconRichText` (though maybe in this case having a slightly different method to directly get the icon path for the fullfilled / missing icon would be fine).

Reply:
I think required is a bit too specific still, since the other use is currently 'Unique' where that wouldn't make sense. Maybe 'CONDITION_SATISFIED' and 'CONDITION_NOT_SATISFIED'?

Reply:
That's true, it wouldn't work there. How about my suggestion about just using that icon I already put in the game? If there's ever a need to "translate" that image, using it in this context as well would get that localization for free for the Thriveopedia as well as that's needed elsewhere in the game already.

Reply:
It looks ok, although I'm not sure green with a white circle is enough to convey 'Yes' in isolation. Next to the red cross it's fine, but on its own the player might assume the circle would be replaced with a tick.

I'll ask on Discord and see what others think.

![image](https://github.com/Revolutionary-Games/Thrive/assets/7430433/2e1f1139-6ce5-429d-983b-bcc5c1c39379)

Reply:
This was brought up before. There wasn't an icon that is universal for the "ok" symbol. I think it is understandable enough given the context and how the player *will* have to understand what the icon means in regards to crafting recipes.

Reply:
I guess icons are fine. Would be a plus to have the "yes" icon localizable though.

Reply:
I've replaced yes/no with icons just to get this moving forward. I would personally still prefer text but we'll see what players think.